src/index.js

```diff
@@ -54,7 +53,7 @@ async function checkQtumd() {
 }
 
 function startQtumProcess(reindex) {
-  const flags = ['-logevents', '-rpcworkqueue=32', '-rpcuser=bodhi', '-rpcpassword=bodhi'];
+  const flags = ['-logevents', '-rpcworkqueue=32', `-rpcuser=${Config.RPC_USER}`, `-rpcpassword=${getRPCPassword()}`];
```

Comment:
for dev, fix credential can mk debug easier

Reply:
it is. for all dev modes i pass in `--rpcpassword` which is used in `getRPCPassword()`. and rpcuser is always `bodhi`